### PR TITLE
Bug 1202190 - Fix dropping icons at end of apps grid. r=gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6170e62acecd7b890f77a020f81ca9a820f3a23b

File: apps/homescreen/js/app.js [hunk 1]
```diff
@@ -654,16 +654,19 @@ const SETTINGS_VERSION = 0;
...
-      // Handle app uninstallation
+      // Handle app/site uninstallation, editing and dragging to the end of
+      // the icon grid.
       case 'drag-end':
-        if (e.detail.clientY <= window.innerHeight - DELETE_DISTANCE) {
+        if ((!this.draggingRemovable && !this.draggingEditable) ||
+            e.detail.clientY <= window.innerHeight - DELETE_DISTANCE) {
           // If the drop target is the container, check to see if we're
           // dropping over the icon itself, and if we aren't, we must be
           // dropping over the end of the container.
```
Comment (gmarty) [inline apps/homescreen/js/app.js]: Does this comment need updating?

Comment (Cwiiis) [inline apps/homescreen/js/app.js]: Ah yes, slight update - cheers.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1acc7b6d41fcc5fc22d8e1db37c1cd955fa28649

**Pull request merged**